src/components/shared/VirtualList.js

```diff
@@ -174,6 +174,7 @@ type VirtualListProps = {|
   +containerWidth: CssPixels,
   +role?: string,
   +ariaLabel?: string,
+  +ariaActiveDescendant?: null | string,
```

Comment:
can you please add some comments for these 3 new props, explaining what they're for? I should have asked earlier for the previous ones :)